tests/h/models/user_identity_test.py

```diff
@@ -101,3 +176,7 @@ def test_repr(self):
             )
 
         assert repr(user_identity) == expected_repr
+
+    @pytest.fixture
```

Comment:
Is there an advantage to making this a fixture versus using the `factories` fixture in the tests that need a user?

Reply:
It's just to allow the tests to take a `user` argument and use it directly, rather than taking a `factories` argument and then having to do `user = factories.User()`.